config/superblock-order.ts

```diff
@@ -0,0 +1,626 @@
+import { Languages } from './i18n';
+import { SuperBlocks } from './certification-settings';
+
+/*
+ * .env SHOW_NEW_CURRICULUM = SuperBlockStates.New
+ * 'New' -> shown only on english staging at the moment
+ *
+ * .env SHOW_UPCOMING_CHANGES = SuperBlockStates.Upcoming
+ * 'Upcoming' is for development -> not shown on stag or prod anywhere
+ *
+ */
+
+export enum CurriculumMaps {
+  Landing = 'landing',
+  Learn = 'learn'
+}
+
+export enum TranslationStates {
+  Audited = 'audited',
+  NotAudited = 'notAudited'
+}
+
+export enum SuperBlockStates {
+  Current = 'current',
+  New = 'new',
+  Upcoming = 'upcoming',
+  Legacy = 'legacy'
+}
+
+export const orderedSuperBlockStates = [
```

Comment:
I'm wondering if this is necessary @ojeytonwilliams. It feels redundant - they are already in the correct order in the enum above. I feel the similarly about `defaultSuperBlockOrder` below, where we could just use the main `SuperBlocks` enum. I didn't with that cause it's in a different file, and I couldn't move it here cause of circular imports. I'm fine leaving these, but they do both feel repetitive. Just curious if you have any thoughts I guess.

Reply:
Right, but the key order is determined by the order they're added to the object.  It's *probably* fine, but I don't like relying on something that feels incidental.